packages/js/src/workouts/tailwind-components/steps/site-representation/site-representation-step.js

```diff
@@ -32,13 +32,15 @@ export default function SiteRepresentationStep( { onOrganizationOrPersonChange,
 		{  window.wpseoWorkoutsData.configuration.knowledgeGraphMessage &&  <Alert type="info">
 			{  window.wpseoWorkoutsData.configuration.knowledgeGraphMessage }
 		</Alert> }
-		<p className="yst-text-sm yst-whitespace-pre-line yst-mb-6">
+		<p className={ classNames( "yst-text-sm yst-whitespace-pre-line yst-mb-6", state.shouldForceCompany ? "yst-mt-2" : "yst-mt-0" ) }>
 			{
 				addLinkToString(
 					sprintf(
 						__(
-							"Help us out here! Is your site about an organization or a person? " +
-							"Completing this step will help Google to understand your website, and improve your chance of getting %1$srich results%2$s.",
+							state.shouldForceCompany
+								? "Completing this step will help Google to understand your website, and improve your chance of getting %1$srich results%2$s."
+								: "Help us out here! Is your site about an organization or a person? " +
+								"Completing this step will help Google to understand your website, and improve your chance of getting %1$srich results%2$s.",
```

Comment:
This syntax prevents the correct extraction of the strings (even the former one with `+` prevented it)